nuxeo-core/nuxeo-core-storage-dbs/src/main/java/org/nuxeo/ecm/core/storage/dbs/DBSSession.java

```diff
@@ -720,11 +741,12 @@ public Document copy(Document source, Document parent, String name) {
         return getDocument(copyState);
     }
 
-    protected String copyRecurse(String sourceId, String parentId, LinkedList<String> ancestorIds, String name) {
+    protected String copyRecurse(String sourceId, String parentId, LinkedList<String> ancestorIds, String name, boolean excludeSpecialChildren) {
         String copyId = copy(sourceId, parentId, ancestorIds, name);
         ancestorIds.addLast(copyId);
-        for (String childId : getChildrenIds(sourceId)) {
-            copyRecurse(childId, copyId, ancestorIds, null);
+        for (String childId : getChildrenIds(sourceId, excludeSpecialChildren)) {
+            // the special flag is not passed recursively
+            copyRecurse(childId, copyId, ancestorIds, null, true);
```

Comment:
If we don't want to exclude them then shouldn't this be `false` ?

Reply:
cf [https://github.com/nuxeo/nuxeo/pull/3375#discussion_r324751037]
cf [(https://github.com/nuxeo/nuxeo/pull/3375#discussion_r324757333)]